### PR TITLE
Fix/load more expense comments

### PR DESCRIPTION
**Pull request by jzabala**
Must be run with backend: https://github.com/opencollective/opencollective-api/pull/2869

Closes https://github.com/opencollective/opencollective/issues/2590

### CODE REVIEW

**Comment by jzabala** [comment]
Is there a way to specify the backend for now deployment?

**Comment by Betree** [comment]
> Is there a way to specify the backend for now deployment?

@jzabala Yes, by using the same branch name for API and Frontend CI will automatically pull it.

**Comment by jzabala** [comment]
> Edit: My bad, that won't work for now deployment, it's only for E2E tests. We don't have deploy previews in place for the API.

Ok. Got it. I was hopping to try it in the automatic `now` deployment. Thanks 🙂
@Betree @znarf Ready to go.

File: lib/initClient.js [hunk 1]
```diff
@@ -63,12 +63,23 @@ function createClient(initialState, graphqlUrl) {
...
-
-  const link = ApolloLink.from([errorLink, authLink.concat(httpLink)]);
+  const apiV2Link = new HttpLink({
+    uri: graphqlUrl || getGraphqlUrl('v2'),
+    fetch,
+  });
+  /** Depending on the value of the context.apiVersion we choose to use the link for the api
+   * v1 or the api v2.
+   */
+  const httpLink = ApolloLink.split(
```
Comment (znarf) [inline lib/initClient.js]: Nice feature.

**Comment by znarf** [comment]
I also have a concern about Apollo internal storage/caching, what's preventing to have conflicts between v1 and v2 objects?

**Comment by jzabala** [comment]
> I also have a concern about Apollo internal storage/caching, what's preventing to have conflicts between v1 and v2 objects?

[Apollo data normalization](https://www.apollographql.com/docs/react/caching/cache-configuration/#data-normalization) is by typename and id. As long as those are different we won't have conflicts. For example the expense id coming from v2 is encoded so, it is stored as a separate entity than the one coming from v1.

**Comment by znarf** [comment]
I tested and it worked as expected!

**Comment by jzabala** [comment]
> Looks good. Rebasing needed to solve conflicts. Deploying the API on staging so we can test.

Rebased and updated to use comment crud from api v2.

**Pull request merged**